api/src/Modules/registration/Controller/Ticket/PostTicket.php

```diff
@@ -0,0 +1,96 @@
+<?php declare(strict_types=1);
+/*.
+    require_module 'standard';
+.*/
+
+namespace App\Modules\registration\Controller\Ticket;
```

Comment:
Is there a reason `registration` is in lowercase, throughout? We've been using sentence case I thought, mostly.

Reply:
registration is lower case because the module name need to match the module name in /modules/ in the base of the source.  This connects the API implementation with the feature module.  The php modules in /modules are all lowercase so that ended up having this lower case.  Now that could be fixed with a bit of case insensitivity or selective case insensitivity if we wanted to add that complexity.